tools/src/main/java/com/nedap/archie/rules/evaluation/AssertionsFixer.java

```diff
@@ -83,7 +85,6 @@ public AssertionsFixer(RuleEvaluation<?> evaluation, RMObjectCreator creator) {
     }
 
 
-
     private void constructMissingStructure(Archetype archetype, String pathOfParent, String lastPathSegment, List<Object> parents) throws XPathExpressionException {
```

Comment:
```suggestion
    private void constructMissingStructure(Archetype archetype, String pathOfParent, String lastPathSegment, List<Object> parents) {
```
This doesn't throw XPathExpressionExceptions anymore.